data/json/itemgroups/magazines.json

```diff
@@ -236,6 +237,8 @@
     "items": [
       ["ak74mag", 10],
       ["makarovmag", 10],
+      ["ppsh41mag", 10],
+      ["ppsh41drum", 10],
```

Comment:
I kept the spawn rate at 10 because

1. every other entry has a spawn rate of 10.

2. the drum was the original magazine for the gun.